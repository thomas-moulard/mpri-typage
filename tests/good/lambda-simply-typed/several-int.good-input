val x = 0
val y = 1
