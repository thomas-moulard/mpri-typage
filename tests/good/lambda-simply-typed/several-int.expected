x = 0

y = 1

y : int
x : int
